Java/core/src/main/java/com/amazon/randomcutforest/tree/INodeView.java

```diff
@@ -41,4 +41,6 @@
 
     double probailityOfSeparation(float[] point);
 
+    int getLeafPointIndex();
```

Comment:
Documentation: for a leaf node, return the index in the point store for the leaf point. If this method is invoked on a non-leaf node then it should throw an IllegalStateException.

Reply:
Done

Reply:
Done